src/components/UserForm/BasicInformationSection.js

```diff
@@ -27,6 +43,10 @@ const BasicInformationSection = React.memo(
         databaseLanguageOptions,
         currentUserId,
     }) => {
+        const { displayEmailVerifiedStatus, error } = useFeatureToggle()
+        const { data: enforceVerifiedEmail, loading: enforceLoading } =
+            useDataQuery(query)
+        console.log(error, 'error')
```

Comment:
remove?
useFeatureToggle does not return an error either, so I'm not sure if you meant to get the potential error from `useDataQuery`?